ArcGISRuntimeSDKQt_QMLSamples/Maps/CreateAndSaveMap/CreateAndSaveMap.qml

```diff
@@ -187,4 +191,11 @@ Rectangle {
         // Return the Map
         return map;
     }
+
+    Component.onCompleted: {
+        // We store and unset the API key from the ArcGISRuntimeEnvironment here to ensure this sample works within the Qt Sample Viewer.
+        // In a standalone app, the API key can be set directly on the basemaps and never onto the Runtime environment.
+        apiKey = "AAPKc25504c4ceaa494b9d161113dab3fc38KIAA-QZy8lNVpp3K2rqUAgZGnaKWXKzXFzONkenePYcPyEnPzWmqk6vLxkBpx5sI"; //ArcGISRuntimeEnvironment.apiKey;
```

Comment:
presumably this shouldn't be checked in?

Reply:
It certainly should not. We'll consider this stolen and generate a new samples testing api key, and I'll update the git commit check to warn against anything starting with "AAPK"